shared/src/main/scala/pl/metastack/metatime/Time.scala

```diff
@@ -64,6 +93,13 @@ trait Hour extends Period with Component {
       case other: Hour => other.h == h
       case _ => false
     }
+	
+  def <(that: Hour): Boolean = {
```

Comment:
Instead of overriding `<` you should implement [`Ordering[T]`](http://www.scala-lang.org/api/2.11.7/index.html#scala.math.Ordering).